content/en/docs/concepts/storage/volume-attributes-classes.md

```diff
@@ -0,0 +1,119 @@
+---
+reviewers:
+- msau42
+- xing-yang
+title: Volume Attributes Classes
+content_type: concept
+weight: 40
+---
+<!-- overview -->
+
+This page assumes that you are familiar with [StorageClasses](/docs/concepts/storage/storage-classes/),
+[volumes](/docs/concepts/storage/volumes/) and [PersistentVolumes](/docs/concepts/storage/persistent-volumes/)
+in Kubernetes.
+
+<!-- body -->
+
+## Introduction
+
+A VolumeAttributesClass provides a way for administrators to describe the mutable
+"classes" of storage they offer. Different classes might map to different quality-of-service levels.
+Kubernetes itself is unopinionated about what these classes represent.
+
+
+## The VolumeAttributesClass Resource
```

Comment:
```suggestion
## The VolumeAttributesClass API
```

Reply:
This is in consistent with the existing Storage Class site? Do we need to change the Storage Class page as well?

Reply:
On the storage class page we call this session ## The StorageClass Resource

Reply:
It'd be nice to improve the docs about StorageClasses, but that can happen in a separate PR.

Reply:
Changed both to use ``API``